src/main/resources/templates/ama.html

```diff
@@ -1,28 +1,41 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
 <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"/>
     <meta charset="UTF-8"/>
     <title th:text="${post.title}">This is a title</title>
 </head>
 <body>
-<a href="/">Home</a>
-<h1 th:text="${post.title}">AMATitle</h1>
-<h2><p th:text="${post.text}">amadescription</p></h2>
-<form action="#" th:action="@{/posts/__${submission}__}" th:object="${commentPost}" method="post">
-    <p><textarea rows="4" cols="50" th:field="*{text}">Ask your question...</textarea></p>
-    <p><input type="submit" value="Submit" /> <input type="reset" value="Reset" /></p>
-</form>
-<h2>Comments</h2>
-<!--/*@thymesVar id="baseComments" type="java.lang.Iterable"*/-->
-<div th:each="comment : ${baseComments}">
-    <p><a th:text="${comment.text}">Title ...</a>
-    <a th:href="@{/comments/upvote(id=${comment.id})}">Upvote</a><span th:text="|(${comment.getUpvotes()})|"/> <a th:href="@{/comments/downvote(id=${comment.id})}">Downvote</a><span th:text="|(${comment.getDownvotes()})|"/>
-    </p>
-    <form th:action="@{/comments/__${comment.id}__/reply}" method="post">
-        <p><input type="text" name="reply" /><input type="submit" value="Reply" /></p>
+<div class="container-fluid">
```

Comment:
It means it will fill the entire available space as opposed to having a fixed width.